controllers/limitador_controller.go

```diff
@@ -163,11 +151,57 @@ func (r *LimitadorReconciler) reconcileLimitadorDeployment(ctx context.Context,
 	return nil
 }
 
+func (r *LimitadorReconciler) reconcileService(ctx context.Context, limitadorObj *limitadorv1alpha1.Limitador) error {
+	logger, err := logr.FromContext(ctx)
+	if err != nil {
+		return err
+	}
```

Comment:
Maybe we can pass the `logger` instance to `reconcileDeployment`, `reconcileService` and `reconcileConfigMap`

Reply:
(just to DRY it up)

Reply:
nah, forget it.